python/paddle/fluid/contrib/int8_inference/README.md

```diff
@@ -0,0 +1,69 @@
+    exe=exe, # required, executor
+    output=int8_model, # required, INT8 model
+    feed_var_names=feed_dict, # required, feed dict
+    fetch_list=fetch_targets) # required, fetch targets
+```
+
+* Call the calibrator.sample_data() after executor run.
+```python
+_, acc1, _ = exe.run(
+    program,
+    feed={feed_dict[0]: image,
+          feed_dict[1]: label},
+    fetch_list=fetch_targets)
+
+calibrator.sample_data() # Step 2
+```
+
+* Call the calibrator.save_int8_model() after sampling over specified iterations (e.g., iterations = 50)
+```python
+calibrator.save_int8_model() # Step 3
+```
+
+## 2. How to run INT8 model
+You can load INT8 model by load_inference_model [API](https://github.com/PaddlePaddle/Paddle/blob/8b50ad80ff6934512d3959947ac1e71ea3fb9ea3/python/paddle/fluid/io.py#L991) and run INT8 inference similar as [FP32](https://github.com/PaddlePaddle/models/blob/develop/fluid/PaddleCV/object_detection/eval.py "FP32").
+
+```python
+[infer_program, feed_dict,
+    fetch_targets] = fluid.io.load_inference_model(model_path, exe)
+```
+
+## 3. Result
+We provide the results of accuracy and performance, measurd on [Intel® Xeon® Platinum Gold Processor](https://ark.intel.com/products/120489/Intel-Xeon-Gold-6148-Processor-27-5M-Cache-2-40-GHz- "Intel® Xeon® Gold 6148 Processor") (also known as Intel® Xeon® Skylake6148).
+
+| Model  | Dataset  | FP32 Accuracy  | INT8 Accuracy  | Accuracy Diff  |
+| ------------ | ------------ | ------------ | ------------ | ------------ |
+| ResNet-50  | Small  | 72.00%  | 72.00%  |  0.00% |
+| MobileNet-V1  | Small  | 62.00%  | 62.00%  | 0.00%  |
+| ResNet-50  | Full ImageNet Val  |  76.63%  | 76.14%  | 0.48% |
+| MobileNet-V1 | Full ImageNet Val  | 70.78%  | 70.41%  | 0.37%  |
+
+Please note that [Small](http://paddle-inference-dist.cdn.bcebos.com/int8/calibration_test_data.tar.gz "Small") is a subset of [full ImageNet validation dataset](http://www.image-net.org/challenges/LSVRC/2012/nnoupb/ILSVRC2012_img_val.tar "full ImageNet validation dataset"). Here is the typical dataset structure, similar to the requirement of [PaddlePaddle models](https://github.com/PaddlePaddle/models/tree/develop/fluid/PaddleCV/image_classification/data "PaddlePaddle models"):
+
+```bash
+$ ls data/ILSVRC2012/
+val  val_list.txt
+```
+
+| Model  | Batch Size  | FP32 Throughput (Images/second)  | INT8 Throughput (Images/second)  | Improvement  |
+| ------------ | ------------ | ------------ | ------------ | ------------ |
+| ResNet-50  | 1  |   14  | 23  | 1.59X |
+| MobileNet-V1 | 1  | 60  | 97  | 1.61X  |
+
+Notes:
+* The accuracy measurement requires the model with `label`.
+* The performance is measurd on SKX6148 (batch size = 1 and iterations = 50). INT8 performance improvement is ~1.33X on Intel® Xeon® Skylake Server ([Reference](https://software.intel.com/en-us/articles/lower-numerical-precision-deep-learning-inference-and-training "Reference")).
```

Comment:
> INT8 performance improvement is ~1.33X

Where is this data 1.33x from?

Reply:
3 instructions to give 4X throughput, 4/3, you may refer to the reference Section 3: image with 1-2-3.

Reply:
You also find the sentence: "This allows for 4x more input at the cost of 3x more instructions or 33.33% more compute."

Reply:
> You also find the sentence: "This allows for 4x more input at the cost of 3x more instructions or 33.33% more compute."

This sentence is hard for user to find it.

@tensor-tang How do you think this sentence "INT8 performance improvement is ~1.33X on Intel® Xeon® Skylake Server "?

Reply:
Maybe it's not solid for using Skylake Server here? I thought this number should be a theoretical value，not tested data on all Skylake Server,  right? 

Maybe you cloud just copy the src sentence from reference, then adding some your explanation here should be easier to understand.

Reply:
Yes, it is theoretical speedup. How about changing to "INT8 theoretical speedup is ~1.33X on Intel® Xeon® Skylake Server (please refer to 'This allows for 4x more input at the cost of 3x more instructions or 33.33% more compute' in the white paper [link])"? I cannot claim more since it is theoretical and it is impossible to test all SKX servers.

Reply:
Done